backtracking knight tour problem/main.cpp

```diff
@@ -0,0 +1,60 @@
+#include <bits/stdc++.h>
```

Comment:
do the same as above, move the file inside that directory. name the directory backtracking_algorithms.